### PR TITLE
Change logo, example images and form link

### PR DESCRIPTION
**Pull request by candidogutierrez**

### CODE REVIEW

**Review by pvisan — APPROVED** [review]

File: _includes/analytics-gain.html [hunk 1]
```diff
@@ -8,17 +8,17 @@ <h4 class="font-weight-light">
...
                 <li>Aprende del mercado en tiempo real</li>
                 <li>Consigue conclusiones accionables</li>
                 <li>Mejora la efectividad en ventas</li>
             </ul>
             <div>
-                <a href="{{ site.analytics_contact_url }}" class="btn btn-lg btn-primary"
-                   data-aos="zoom-out" data-aos-easing="ease-in-sine" data-aos-duration="1000"
-                >contacta con nosotros</a>
+                <a href="{{ site.analytics_form_url }}" class="btn btn-lg btn-primary"
+                   data-aos="zoom-out" data-aos-easing="ease-in-sine" data-aos-duration="1000" target="_blank">
```
Comment (durey) [inline _includes/analytics-gain.html]: This is widely considered a very bad practice... You are forcing the user to open a tab when the decision should be his

File: _sass/frogtek-theme.scss [hunk 2]
```diff
@@ -420,6 +420,10 @@ footer {
   text-transform: lowercase;
 }
 
+.center-div {
```
Comment (durey) [inline _sass/frogtek-theme.scss]: This is widely considered a very bad practice... You shouldn't name Css classes related to the actual style they apply.

File: _includes/analytics-features.html [hunk 3]
```diff
@@ -3,7 +3,7 @@
     <div class="container-fluid">
         <div class="row justify-content-center bg-dark text-white">
             <div class="col col-md-8 p-1 p-md-4 text-center display-4">
-                con nuestros datos apoyamos el crecimiento de tu negocio
+                Con nuestros datos apoyamos el crecimiento de tu negocio
```
Comment (durey) [inline _includes/analytics-features.html:6]: ❤️

File: _includes/analytics-gain.html [hunk 4]
```diff
@@ -8,17 +8,17 @@ <h4 class="font-weight-light">
                 <p>Grandes empresas ya se benefician de datos obtenidos de
                     <span class="analytics-brand">Frogtek Analytics</span></p>
             </h4>
-            <p class="display-4">¡no te quedes atrás!</p>
+            <p class="display-4">¡No te quedes atrás!</p>
```
Comment (durey) [inline _includes/analytics-gain.html:11]: ❤️

**Review by durey — COMMENTED** [review]
@candidogutierrez Now all the images weight twice as before... Do you want me to reduce them with a modern operative system :trollface: ?

**Comment by candidogutierrez** [comment]
@durey reduce the images, please

**Comment by candidogutierrez** [comment]
@durey changes made!!

**Comment by durey** [comment]
@candidogutierrez Done, bro! (not your fault, but the images folder is a mess... sizes, names... terrible).

**Comment by candidogutierrez** [comment]
@durey do you think we can merge the changes?

**Review by durey — APPROVED** [review]

**Comment by durey** [comment]
Sure (that's Janitor's material)

**Pull request merged**